content/docs/2.14/scalers/cassandra.md

```diff
@@ -44,8 +44,11 @@ You can authenticate by using a password via `TriggerAuthentication` configurati
 **Password Authentication:**
 
 - `password` - Password for configured user to log in to the Cassandra instance.
+- `tls` - To enable SSL auth for Cassandra session, set this to enable. If not set, TLS for Cassandra is not used. (Values: enable, disable, Default: disable, Optional).
+- `cert` - Certificate path for client authentication. Mandatory is tls enabled. (Optional)
+- `key` - Key path for client authentication. Mandatory is tls enabled. (Optional)
```

Comment:
```suggestion
- `cert` - Certificate path for client authentication. Mandatory if `tls` is enabled. (Optional)
- `key` - Key path for client authentication. Mandatory if `tls` is enabled. (Optional)
```

Reply:
@zroubalik 
Thank you for pointing the mistake. I have corrected.